test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { baseLogger } from "src/logger"
+import { btc2sat } from "src/utils"
+import { MainBook } from "src/mongodb"
+import { liabilitiesReserve, lndAccountingPath } from "src/ledger/ledger"
+import {
+  checkIsBalanced,
+  lnd1,
+  lndOutside1,
+  bitcoindClient,
+  getChainBalance,
+  fundLnd,
+  waitUntilSyncAll,
+} from "test/helpers"
+
+jest.mock("src/realtimePrice", () => require("test/mocks/realtimePrice"))
+
+const defaultWallet = ""
+
+beforeEach(async () => {
+  await waitUntilSyncAll()
+})
+
+describe("Bitcoind", () => {
+  it("should be funded mining 10 blocks", async () => {
+    const numOfBlock = 10
+    try {
+      const { name } = await bitcoindClient.createWallet(defaultWallet)
```

Comment:
this could be its own test: "create default wallet"

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/340/commits/7eba5e062a25234804eac0b372da682cff2a8ce2